server/schema/submissions/createSubmission.js

```diff
@@ -0,0 +1,24 @@
+const schema = {
+  required: ['user_submission', 'output'],
```

Comment:
```suggestion
  required: ['user_submission'],
```
In code problems, user will just submit his code, nothing else. Output will be found by us, by running the code